connector/routingconnector/internal/fanoutconsumer/logs.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package fanoutconsumer contains implementations of Traces/Metrics/Logs consumers
+// that fan out the data to multiple other consumers.
+package fanoutconsumer // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/routingconnector/internal/fanoutconsumer"
```

Comment:
Is this a copy of what we have at the fanoutconsumer in the core? If so, explaining why this was copied and what was changed would be good.

Reply:
I just saw the mentions to this in the PR description. I think it needs to be in the code as well. Like: 

```
TODO: once we have fanoutconsumer test helpers, use that instead of copying the code
```